pkg/ingester/ingester.go

```diff
@@ -81,6 +87,10 @@ func newIngesterMetrics(r prometheus.Registerer) *ingesterMetrics {
 			// A small number of chunks per series - 10*(8^(7-1)) = 2.6m.
 			Buckets: prometheus.ExponentialBuckets(10, 8, 7),
 		}),
+		walReplayDuration: prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
I think this metric is not registered. You should register it below.

Reply:
Oops, thanks!